tests/test_internals/test_tools.py

```diff
@@ -26,3 +29,13 @@ def test_get_module(self):
         assert isinstance(mod_2, ProcessingModule)
         assert mod_2.description == description_2
         self.assertWarns(UserWarning, get_module, **dict(nwbfile=nwbfile, name=name_1, description=description_2))
+
+    def test_make_nwbfile_from_metadata(self):
+        with self.assertRaisesWith(
+            exc_type=AssertionError,
+            exc_msg=(
+                "'session_start_time' was not found auto-populated in metadata['NWBFile']! "
```

Comment:
```suggestion
                "'session_start_time' was not found in metadata['NWBFile']! "
```
it doesn't necessarily need to be auto-populated